### DISCUSSION TITLE
【项目组队】发布参考模板

### DISCUSSION THREAD

**Discussion by ShirleneLiu** [creatorsand/.github#60] [Ideas]
**一句话 IDEA：**

简明扼要地描述项目的核心创意或目标。

**我的角色：**

你在项目中的具体角色和负责的任务。

**其他角色需求（不包含发起人）：**

项目需要的其他角色或技能，发起人以外的团队成员需求。

**社群昵称：**

你在社群中的昵称或身份。

**个人简介：**

简短介绍你的背景、经验及相关技能。

**产品是给谁用的？**

目标用户群体是什么，或者解决什么特定用户群的需求。

**其他产品有什么痛点，用户为什么要选择你的产品？**

当前市场上其他产品存在的痛点是什么，以及你的产品如何独特地解决这些问题，吸引用户选择。

**Comment by tongsiyuan-nm2207** [comment]


**Reply by ShirleneLiu** [reply]
同步项目需要单开一个 discussion 😙
